3rdParty/fuerte/src/http_parser/http_parser.c

```diff
@@ -151,7 +151,7 @@ do {                                                                 \
  */
 #define COUNT_HEADER_SIZE(V)                                         \
 do {                                                                 \
-  nread += (V);                                                      \
+  nread += (uint32_t)(V);                                            \
```

Comment:
Is this the correct place? Now anything could be put in the macro and we would not know what is going wrong,